S/Spot_julia/build_tarballs.jl

```diff
@@ -3,12 +3,12 @@
 using BinaryBuilder, Pkg
 
 name = "Spot_julia"
-version = v"2.9.7"
+version = v"2.12"
 julia_versions = [v"1.6.3", v"1.7", v"1.8", v"1.9", v"1.10"]
```

Comment:
While you are it, why not update your package to drop this hard coded list, so it'll be compatible with 1.11.0 on the day it is released, and also with 1.12 dev versions?

To do so, it should be sufficient to drop this line.
```suggestion
```

Reply:
Thank you, I don't remember why I had this.